dtk/process.py

```diff
@@ -525,9 +532,82 @@ def nextpow2(i):
     except IndexError:
         amplitude = 2*abs(Y[1:n//2])
         # power = abs(Y[1:n/2])**2
+
     return frequency, amplitude
 
 
+def pow_spectrum(data, sampleRate):
```

Comment:
Let's follow pep8 naming conventions for new things (for some reason I didn't follow it before.

Reply:
`sample_rate`

Reply:
I fully agree. I only kept this one for consistency with ```freq_spectrum()```